docs/pages/server-access/guides/ec2-discovery.mdx

```diff
@@ -0,0 +1,308 @@
+Discovery Node must include the `AmazonSSMManagedInstanceCore` IAM policy
+in order to receive commands from the Discovery Node. 
+
+This policy includes the following permissions:
+```js
+{
+    "Version": "2012-10-17",
+    "Statement": [
+        {
+            "Effect": "Allow",
+            "Action": [
+                "ssm:DescribeAssociation",
+                "ssm:GetDeployablePatchSnapshotForInstance",
+                "ssm:GetDocument",
+                "ssm:DescribeDocument",
+                "ssm:GetManifest",
+                "ssm:GetParameter",
+                "ssm:GetParameters",
+                "ssm:ListAssociations",
+                "ssm:ListInstanceAssociations",
+                "ssm:PutInventory",
+                "ssm:PutComplianceItems",
+                "ssm:PutConfigurePackageResult",
+                "ssm:UpdateAssociationStatus",
+                "ssm:UpdateInstanceAssociationStatus",
+                "ssm:UpdateInstanceInformation"
+            ],
+            "Resource": "*"
+        },
+        {
+            "Effect": "Allow",
+            "Action": [
+                "ssmmessages:CreateControlChannel",
+                "ssmmessages:CreateDataChannel",
+                "ssmmessages:OpenControlChannel",
+                "ssmmessages:OpenDataChannel"
+            ],
+            "Resource": "*"
+        },
+        {
+            "Effect": "Allow",
+            "Action": [
+                "ec2messages:AcknowledgeMessage",
+                "ec2messages:DeleteMessage",
+                "ec2messages:FailMessage",
+                "ec2messages:GetEndpoint",
+                "ec2messages:GetMessages",
+                "ec2messages:SendReply"
+            ],
+            "Resource": "*"
+        }
+    ]
+}
+```
+
+## Step 4/5. Configure Teleport to discover EC2 instances
+
+The discovery service requires a valid auth token to connect to the cluster. Generate one by running the following command against your Teleport Auth Service and save it in `/tmp/token` on the node that will run the discovery service:
+
+```code
+$ tctl tokens add --type=discovery
+```
+
+In order to enable EC2 instance discovery the `discovery_service.aws` section must
+include at least one entry:
+
+```yaml
+version: v2
+teleport:
+  join_params:
+    token_name: "/tmp/token"
+    method: token
+  auth_servers:
+  - "teleport.example.com:3080"
+auth_service:
+  enabled: off
+proxy_service:
+  enabled: off
+ssh_service:
+  enabled: off
+discovery_service:
+  enabled: "yes"
+  aws:
+   - types: ["ec2"]
+     regions: ["eu-central-1"]
+     tags:
+       "env": "prod" # Match EC2 instances where tag:env=prod
+```
+
+Once Teleport is configured for discovery, it can be started and EC2
+instances matching the tags specified in the AWS section will begin to
+be added to the Teleport cluster automatically.
+
+## Step 5/5. [Optional] Customize the default installer script
+
+To customize the default installer script, execute the following command:
```

Comment:
```suggestion
To customize the default installer script, execute the following command on your workstation:
```
We mention a few different hosts in this page, so we should indicate which one you should use to run a given command.